### PR TITLE
Scheduled protocol upgrade blogpost

### PR DESCRIPTION
**Pull request by dEBRUYNE-1**

### CODE REVIEW

**Comment by MaxXor** [comment]
Although I agree that such a blogpost is important, I think it's pretty short term before the hardfork.

File: _posts/2017-09-13-a-scheduled-protocol upgrade-is-planned for-September-15-2017-09-13.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+https://getmonero.org/downloads/  
+
+--------------  
+
+**Note that this is a scheduled and consensual protocol upgrade. Thus, unlike the BCH / BTC split, a new coin *won't* be created.**  
+
+-------------  
+
+Approximately the 15th of September there will be a scheduled protocol upgrade on the Monero network. Most pools have upgraded or are in the process of upgrading, but some have not upgraded yet. If they don't upgrade before the scheduled protocol upgrade they will get booted off the network. As a result you will miss out on revenue if you are mining on these pools. Thus, if you are mining on one of the pools that hasn't upgraded yet or hasn't scheduled an upgrade, please contact your pool owner as soon as possible and urge them to upgrade. Alternatively, you can switch to a pool that is on the right version.
```
[the remark this answers is not in the document]
Comment (dEBRUYNE-1) [reply _posts/2017-09-13-a-scheduled-protocol upgrade-is-planned for-September-15-2017-09-13.md]: Thanks, fixed.

File: _posts/2017-09-13-a-scheduled-protocol upgrade-is-planned for-September-15-2017-09-13.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+A list of pools and services and their respective statuses can be found [here](https://www.reddit.com/r/Monero/comments/6yuv2s/psa_users_solominers_exchangesmerchantsservices/).  
+
+# An important message for pool operators and miners  
+
+Bear in mind that running CLI v0.11.0.0 or GUI v0.11.0.0 is mandatory. Any other versions will get booted off the network. Thus, miners, please email your pools and ask them if they are running v0.11.0.0. In addition, pool operators need to ensure that payouts use a minimum ring size of 5 (mixin 4), otherwise they will get rejected by the network.  
+
+# General scheduled protocol upgrade information  
+
+The scheduled protocol upgrade was introduced to enforce RingCT transactions and to bump the minimum ring size to 5 (mixin 4). Note that, after this scheduled protocol upgrade, you are still able to transform non RingCT outputs to RingCT outputs in a "one-time" transaction. Thus, you **don't** need to sweep a paper wallet if it contains non RingCT outputs.
```
[the remark this answers is not in the document]
Comment (dEBRUYNE-1) [reply _posts/2017-09-13-a-scheduled-protocol upgrade-is-planned for-September-15-2017-09-13.md]: I am not sure if this is necessarily erroneous.

File: _posts/2017-09-13-a-scheduled-protocol upgrade-is-planned for-September-15-2017-09-13.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+# An important message for pool operators and miners  
+
+Bear in mind that running CLI v0.11.0.0 or GUI v0.11.0.0 is mandatory. Any other versions will get booted off the network. Thus, miners, please email your pools and ask them if they are running v0.11.0.0. In addition, pool operators need to ensure that payouts use a minimum ring size of 5 (mixin 4), otherwise they will get rejected by the network.  
+
+# General scheduled protocol upgrade information  
+
+The scheduled protocol upgrade was introduced to enforce RingCT transactions and to bump the minimum ring size to 5 (mixin 4). Note that, after this scheduled protocol upgrade, you are still able to transform non RingCT outputs to RingCT outputs in a "one-time" transaction. Thus, you **don't** need to sweep a paper wallet if it contains non RingCT outputs.   
+
+Due to variance the scheduled protocol upgrade will take place on or around the 15th of September. There is, however, no specific date as the hard fork will be triggered by a specific blockheight. The specific blockheight for the hardfork can be found [here](https://github.com/monero-project/monero/blob/master/src/cryptonote_core/blockchain.cpp#L105). That is:
```
[the remark this answers is not in the document]
Comment (dEBRUYNE-1) [reply _posts/2017-09-13-a-scheduled-protocol upgrade-is-planned for-September-15-2017-09-13.md]: Thanks fixed & same comment as above.

**Comment by dEBRUYNE-1** [comment]
@MaxXor, regardless, it'll be good for visibility. If users are booted off the network, they will see this blog post once they browse to the website to see what's going on.

File: _posts/2017-09-13-a-scheduled-protocol upgrade-is-planned for-September-15-2017-09-13.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (QuickBASIC) [reply _posts/2017-09-13-a-scheduled-protocol upgrade-is-planned for-September-15-2017-09-13.md]: I thought there was an extra comma, but after re-reading it seems fine. Sorry about that.

**Pull request merged**